mmv1/third_party/terraform/services/compute/compute_instance_helpers.go.erb

```diff
@@ -387,6 +406,11 @@ func expandAccessConfigs(configs []interface{}) []*compute.AccessConfig {
 				acs[i].SetPublicPtr = true
 				acs[i].PublicPtrDomainName = ptr.(string)
 			}
+			<% unless version == 'ga' -%>
+			if sp, ok := data["security_policy"]; ok && sp != "" {
+				acs[i].SecurityPolicy = sp.(string)
+			}
+			<% end -%>
```

Comment:
As `security_policy` in the access config is computed, is this block needed?

Reply:
Removed.